submissions/dashakim/memory-game/script.js

```diff
@@ -0,0 +1,90 @@
+const section = document.querySelector("section");
+
+const imagesNames = function(currentImg) {
+  return currentImg
+    .split(".")
+    .slice(0, 1)
+    .join("")
+    .substr(4);
+};
+
+const createFrontFaceCard = function(div) {
+  const frontImage = document.createElement("img");
+  frontImage.classList.add("front-face");
+  frontImage.setAttribute("src", frontImageQuestion);
+  frontImage.setAttribute("alt", "icon-question");
+  div.appendChild(frontImage);
+};
+
+const createBackFaceCard = function(div, el) {
+  const backImage = document.createElement("img");
+  backImage.classList.add("back-face");
+  backImage.setAttribute("src", el);
+  backImage.setAttribute("alt", imagesNames(el));
+  div.appendChild(backImage);
+};
+
+const createCard = function(storage) {
+  storage.map(i => {
```

Comment:
Some things that raise questions here:
1. Function iterates through an array and may create more than one card. So we can rename our function or take another way: move array enumeration beyond function and call that function as a callback in `.map` method.
2. [As we know](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) - calling `map` method returns new array. Is it really important for us in this case, or we could use another array method without redundant functionality?
3. map callback function taking three arguments: element, index, array (in described order). We can name parameters as we want, but due to some conventions more often use **'e'** for `element` and **'i'** for `index` shortcut.

Reply:
Callbacks it's my weak side. I'm not sure how rewrite it, so I didn't fix it